Mathlib/Analysis/SpecialFunctions/Log/Base.lean

```diff
@@ -457,10 +481,87 @@ theorem logb_nat_eq_sum_factorization (n : ℕ) :
     logb b n = n.factorization.sum fun p t => t * logb b p := by
   simp only [logb, mul_div_assoc', log_nat_eq_sum_factorization n, Finsupp.sum, Finset.sum_div]
 
--- TODO add other limits and continuous API lemmas analogous to those in Log.lean
+theorem tendsto_pow_logb_div_mul_add_atTop_aux (a c : ℝ) (ha : a ≠ 0) :
+    Tendsto (fun x => (a * x + c)⁻¹) atTop (𝓝 0) := by
+  obtain ha' | ha' := lt_or_gt_of_ne ha
+  · exact tendsto_inv_atBot_zero.comp
+      (tendsto_atBot_add_const_right _ c (tendsto_id.const_mul_atTop_of_neg ha'))
+  · exact tendsto_inv_atTop_zero.comp
+      (tendsto_atTop_add_const_right _ c (tendsto_id.const_mul_atTop ha'))
```

Comment:
This lemma shouldn't be in this file, but I'd like suggestions on where it should go instead.
(I'll change the name and move it at the same time)

Reply:
This looks like it should go somewhere in `Analysis/SpecificLimits`.